cli/utils.py

```diff
@@ -2,17 +2,20 @@
 import pathlib
 import subprocess
 from datetime import datetime
+from io import BytesIO
 
 import click
+import git
 import structlog
 
 from covidactnow.datapublic import common_df
 from covidactnow.datapublic.common_fields import CommonFields
 from libs import github_utils
-from libs.datasets import combined_datasets
+from libs.datasets import combined_datasets, dataset_utils
```

Comment:
Can you add this import on a separate line? recommended practice by the [gsg](https://google.github.io/styleguide/pyguide.html#313-imports-formatting)